1-exercises/D-for-in-loops/exercise-1.js

```diff
@@ -14,6 +14,14 @@ let salaries = {
 
 // WRITE BELOW THIS LINE
 
+for (let name in salaries) {
+  //console.log(name); // ross monica rachel
+  let personName = salaries[name]; // value
+  if (personName > 50000) {
```

Comment:
Suggestion on the variable naming. Line 19 **personName** is actually **personSalary**